client/src/document/toolbar/flaws.tsx

```diff
@@ -1064,3 +1072,41 @@ function UnsafeHTML({ flaws }: { flaws: UnsafeHTMLFlaw[] }) {
     </div>
   );
 }
+
+function TranslationDifferences({
+  flaws,
+}: {
+  flaws: TranslationDifferenceFlaw[];
+}) {
+  return (
+    <div className="flaw">
+      <h3>{humanizeFlawName("translation_differences")}</h3>
+      <ul>
+        {flaws.map((flaw, i) => {
+          const key = flaw.id;
+          let explanation = <b>{flaw.explanation}</b>;
+
+          return (
+            <li key={key}>
+              {explanation}
+              {flaw.difference.fullExplanation &&
+                flaw.difference.fullExplanation.length > 0 && (
+                  <ul className="full-explanation">
+                    {flaw.difference.fullExplanation.map(
+                      (fullExplanation, i) => {
+                        return (
+                          <li key={`${fullExplanation}${i}`}>
+                            <code>{fullExplanation}</code>
+                          </li>
+                        );
+                      }
+                    )}
+                  </ul>
+                )}
+            </li>
+          );
+        })}
```

Comment:
nit: imo inlining the 1-time-use vars and using short-hand arrow fns makes for more readable code here
```suggestion
        {flaws.map((flaw) => (
          <li key={flaw.id}>
            <b>{flaw.explanation}</b>
            {flaw.difference.fullExplanation &&
              flaw.difference.fullExplanation.length > 0 && (
                <ul className="full-explanation">
                  {flaw.difference.fullExplanation.map((fullExplanation, i) => (
                    <li key={`${fullExplanation}${i}`}>
                      <code>{fullExplanation}</code>
                    </li>
                  ))}
                </ul>
              )}
          </li>
        ))}
```